jetson/percep/pcl.hpp

```diff
@@ -5,24 +5,44 @@
 #include <pcl/common/common_headers.h>
 
 /* --- Compare Line Class --- */
-//Functor that indicates where a point is in
-//relation to a line in 2D space
+/**
+ \brief Functor that indicates where a point is in
```

Comment:
Why the leading slash? Also, why are you changing the style of this comment? I feel it should be consistent with what is already there. However #516 will hopefully take care of this.

Reply:
I'll add it to Issue #516